aws/scripts/destroy.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2154,SC1091
+
+# Copyright 2021 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Bash safeties: exit on error, no unset variables, pipelines can't hide errors
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Locate the root directory
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+# Run common.sh script for variable declaration and validation
+source "${ROOT}/scripts/common.sh"
+
+# Exporting the bucket name as an environment variable.
+export bucket_name="${TF_VAR_name}-bucket-statefiles"
+
+# Generate Backend Template to store Terraform State files.
+readonly backend_config="terraform {
+  backend \"s3\" {
+    region = \"${TF_VAR_region}\"
+    bucket = \"${bucket_name}\"
+    key    = \"terraform/${TF_VAR_environment}/\"
+  }
+}"
+
+cd "${ROOT}/env"
+echo -e "${backend_config}" > backend.tf
```

Comment:
Should this script exist if we are not using a remote backend in this repo?

Reply:
But we have decided that, we will be storing our remote backends while using the make commands.

Reply:
Did we decide that? I thought we were going to provide a reference for what a remote backend configuration would look like in the README and leave it at that. End users can look up the appropriate Terraform docs _if_ they want to leverage that functionality.